go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/cpuguy83/go-md2man v1.0.10 // indirect
 	github.com/creack/pty v1.1.7
-	github.com/digitalocean/godo v1.22.0
+	github.com/digitalocean/godo v1.26.0
```

Comment:
This is done by Kamal's currently open PR, so once he merges that, you're probably good to rebase against that to get `godo v1.26.0`.

Reply:
When is that getting merged? If all goes well, this may get in before that :smirk: